packages/airnode-deployer/src/handlers/gcp/index.ts

```diff
@@ -114,15 +114,19 @@ export async function processHttpSignedDataRequest(req: Request, res: Response)
     res.status(401).send({ error: 'Wrong API key' });
   }
 
-  const { parameters } = req.body;
+  const { encodedParameters } = req.body;
```

Comment:
Nit: it's kinda weird that here we assign by destructuring but above `aws/index.ts` it's done with regular assignment like `const encodedParameters = JSON.parse(event.body).encodedParameters;`.

I kinda think we should just make it consistent while we're modifying these files